plugins/store/README.md

```diff
@@ -68,9 +68,9 @@ fn main() {
 Afterwards all the plugin's APIs are available through the JavaScript guest bindings:
 
 ```typescript
-import { Store } from '@tauri-apps/plugin-store'
+import { LazyStore } from '@tauri-apps/plugin-store'
```

Comment:
I would like to push people to use `LazyStore` most of the times, what's your thoughts?

Reply:
I usually don't like hidden/magic code, and so `LazyStore` should be sort of opt-in when you understand how it behaves, otherwise I fear we will receive issues that goes like this "I copied this snippet, but the behavior is not as I expected"

Reply:
Showing what `LazyStore` offers after showing `Store` will explain it better and what it fixes or enhances.

Reply:
I think a lazy store is what most people want, I would prefer we document the primitives after the higher level API `LazyStore` in this case

Reply:
Doesn't make sense to show `LazyStore` after `Store`. As a user, I wouldn't understand what `LazyStore` does unless I understand `Store` first.

Reply:
`LazyStore` works exactly like the old `Store`, and it's quite simple to understand, but documenting the primitives also make sense, I'll change it